README.md

```diff
@@ -15,7 +21,7 @@ nagios_graphite, however some of the switches have changed name (both
 long and short) so pay close attention when migrating.
 
 ```shell
-check-graphite -g graphite.example.com -m 'my.metric' -w 10 -c 100
+go-check-graphite -g graphite.example.com -m 'my.metric' -w 10 -c 100
```

Comment:
This slightly confusing the but the executable *as produced by goreleaser* is actually `check-graphite`, but when using `go install` it uses the default executable name which is equal to the last directory in the install path.

Reply:
Oh, interesting, I wasn't aware of that caveat. It was something I ran into so I figured others might end up pinging you with the same question :)